project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def migrate(cr, _):
+    logger.info(
+        "Set project.task.type is_closed field using closed field if set "
+        " and drop closed field"
+    )
+    cr.execute(
+        """UPDATE project_task_type SET is_closed = closed
+            WHERE NOT closed is not NULL;
+        """
+        "ALTER TABLE project_task_type DROP COLUMN closed;"
```

Comment:
Don't remove the column. Odoo already does it.

Reply:
I've been tricked by odoo message

> 2021-05-23 18:52:48,702 30278 WARNING oca-project odoo.addons.base.models.ir_model: Deleting field project.task.type.closed (hint: fields should be explicitly removed by an upgrade script) 

Thanks